scripts.md

```diff
@@ -49,6 +49,12 @@ this repo.
 ### `npm run test`
 This script runs `npm run test` on each of the Blockly plugins in this repo.
 
+### `npm run test:ghpages`
+This script builds all files needed to deploy plugins and examples to Github Pages, then starts a local server with that content.
+
+### `npm run test:ghpages:beta`
+This script installs a beta version of Blockly, builds all files needed to deploy plugins and examples to Github Pages, then starts a local server with that content.
```

Comment:
According to them, the correct capitalization is 'GitHub'.  Here and three lines up.